Arriba/Arriba.Test/Model/ColumnFactoryTests.cs

```diff
@@ -16,6 +15,12 @@ public class ColumnFactoryTests
     {
         private string _defaultWrappingFormatString = "UntypedColumn<{0}>;FastAddSortedColumn<{0}>;ValueTypeColumn<{0}>";
 
+        [TestCleanup]
+        public void TestCleanup()
+        {
+            ColumnFactory.ResetColumnCreators();
```

Comment:
Maybe we should allow removing them individually only. Otherwise, this method seems dangerous outside of text contexts.